GUI/PyQt/framework1.ui

```diff
@@ -6,229 +6,329 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>1340</width>
-    <height>849</height>
+    <width>1796</width>

```

Comment:
Is this file fixed or generate each time automatically?
If fixed: Are the values empirical chosen?